openedx/features/caliper_tracking/settings.py

```diff
@@ -0,0 +1,14 @@
+"""
+All settings for Caliper Tracking app.
+"""
+
+CALIPER_TRACKING_PROCESSOR = 'openedx.features.caliper_tracking.processor.CaliperProcessor'
+
+CALIPER_TRACKING_BACKENDS = {
+    'logger': {
+        'ENGINE': CALIPER_TRACKING_PROCESSOR,
```

Comment:
Why do you need to define a constant if its being referred in the same file?

Reply:
The same constant is also being used in another file.